community/modules/scheduler/schedmd-slurm-gcp-v5-login/variables.tf

```diff
@@ -0,0 +1,284 @@
+/**
```

Comment:
One pattern I see is that it seems that perhaps the underlying slurm module has taken over more intelligent defaults  that we can rely on, but as a side effect the defaults of our module have become more opaque. I am not quite sure what to do about this. Any thoughts?

Maybe more prominent linking to their documentation for information on defaults?

Reply:
I have added links in a few places to clarify defaults.

Preferably defaults are listed and therefore self-documenting, but in many cases the underlying modules are setting them automatically instead. There isn't really a single solution I can see to this, as different variables behave differently.

Documenting them as we find them, with links as needed, is probably the best approach for now.